packages/design-tokens/design-token-previews/utils/constants.tsx

```diff
@@ -0,0 +1,111 @@
+import fontSizeTokens from "../../src/tokens/font-size.tokens.json";
+import fontWeightTokens from "../../src/tokens/font-weight.tokens.json";
+import colorTokens from "../../src/tokens/color.tokens.json";
+import iconSizeTokens from "../../src/tokens/size.tokens.json";
+import spaceTokens from "../../src/tokens/space.tokens.json";
+import borderStyleTokens from "../../src/tokens/border-style.tokens.json";
+import borderWidthTokens from "../../src/tokens/border-width.tokens.json";
+import borderRadiiTokens from "../../src/tokens/border-radius.tokens.json";
+
+import { TokenEntry } from "../types/TokenEntry";
+
+import { createBox } from "../components/createBox";
+import {
+  createIconSizePreview,
+  createFontSizePreview,
+  getTokenComment,
+  getTokenKey,
+  getTokenName,
+  getTokenValue,
+  hexToRgb,
+  hexToHsla,
+} from "../utils";
+
+const EXAMPLE_TEXT = "The quick brown fox jumped over the lazy dog.";
+
+export const TOKEN_COLUMNS = {
+  [getTokenKey(borderRadiiTokens)]: [
+    {
+      name: "Name",
+      transform: getTokenName(getTokenKey(borderRadiiTokens)),
```

Comment:
I would consider having a square passing the optional props `{ w: "50px", h: "50px"}`, for example. Because we have a circle. 

We would benefit from some margin too as the items seem too close. I left a comment with the screenshot before, but I believe you missed it.

Reply:
I'm not sure about this one. I made them with the height and width but we also have a pill shape and that should not be a square.